templates/bene_404.html

```diff
@@ -58,7 +58,9 @@
           If you are not a new Medicare enrollee, please report this error to <a href="mailto:bluebuttonapi@cms.hhs.gov">bluebuttonapi@cms.hhs.gov</a>.
           Please DO NOT email any sensitive personal information such as date of birth, address, or Medicare numbers.
         </p>
-		  </div>
+        <p>Request ID: {{ request_id }}</p>
```

Comment:
Should probably check with Product about how to incorporate this. We might want to specifically call out "include this request id in your email" or something.